### ISSUE TITLE
Can allure-kotlin work with Junit5?

### ISSUE THREAD

**Issue by Rodion777777** [allure-framework/allure-kotlin#36]
Can allure-kotlin work with Junit5?

**Comment by kamildziadek** [comment]
No, currently it supports only JUnit4. Although, the repository was prepared in such a way that adding support for new tool is just a matter of creating a new module and implementing run listener based on the implementation of it in [allure-java](https://github.com/allure-framework/allure-java)
